### ISSUE TITLE
missing documentation.value in completion items on Linux

### ISSUE THREAD

**Issue by Eskibear** [redhat-developer/vscode-java#923]
The issue was first mentioned by @muggell in https://github.com/redhat-developer/vscode-java/issues/842#issuecomment-491744539

The details are missing in completion items. The issue keeps occurring for @muggell no matter IntelliCode is enabled or disabled.

![image1](https://user-images.githubusercontent.com/32263698/57617081-2cd60d00-7580-11e9-9600-4cbab929d1a5.png)
![image2](https://user-images.githubusercontent.com/32263698/57617111-3fe8dd00-7580-11e9-8373-e6c679e088c6.png)

> While I still get the suggestions, I can't get the "Read more.." information. It just loads the top spar showing the "Math" and nothing below it. It's driving me nuts. I really enjoy VSCode on my Windows Desktop, but I can't seem to get it working on my Linux laptop.
So would you mind sharing your solution? Perhaps that might help me out.

> Edit: I do on the other hand get suggestions and further information when I use VSCode for .js files.
##### Environment
- Operating System: some Linux distribution, see the original issue.
- Java extension version: @muggell please also provide this.

>VSCode version: 1.33.1
IntelliCode extension version: 1.1.6
JDK: openjdk-11 or java-1.11.0-openjdk-amd64

##### Additional Informations
As @muggell mentioned in https://github.com/redhat-developer/vscode-java/issues/842#issuecomment-492179360

```
[Trace - 12:15:40 PM] Received response 'completionItem/resolve - (40)' in 573ms.
Result: {
    "label": "E : double",
    "kind": 21,
    "detail": "Math",
    "documentation": {
        "kind": "markdown"
// ----> HERE THE "value" FIELD IS MISSING
    },
    "sortText": "999999178",
    "insertText": "E",
    "insertTextFormat": 2,
    "textEdit": {
        "range": {
            "start": {
                "line": 9,
                "character": 25
            },
            "end": {
                "line": 9,
                "character": 25
            }
        },
        "newText": "E"
    }
}

```

**Comment by Eskibear** [comment]
When trying to reproduce it in a clean installed Ubuntu 18.04, I myself also encountered the issue for some times, but **only occurs in the first time I trigger completion** after vscode-java is activated.   

@muggell please follow this issue, @yaohaizh @fbricon might be able to help you out.

**Comment by muggell** [comment]
@Eskibear  Thanks for helping out. To provide the last bit of information: 
Linux distro: Pop!_OS 19.04
Java Extension Version: 0.7.1

**Comment by muggell** [comment]
I guess nobody is up for the task @Eskibear or are they just really busy all the time? :-)

**Comment by fbricon** [comment]
@muggell see if https://github.com/eclipse/eclipse.jdt.ls/issues/1006#issuecomment-488875894 is relevant to you

**Comment by muggell** [comment]
Cheers @fbricon it was actually the same thing that was wrong. It did for some reason look up the wrong path for the documentation/sources. So I think that it might be fixed now.

**Comment by ahmedhamdy90** [comment]
> Cheers @fbricon it was actually the same thing that was wrong. It did for some reason look up the wrong path for the documentation/sources. So I think that it might be fixed now.

Can you please elaborate on how to update this path? Knowing that hovering over a function shows the documentation correctly, but it is not showing any on code completion.

**Comment by Ortega-Dan** [comment]
If javadoc shows on hovering but not on autocomplete it should be a different reason, because the hover way is actually able to find the autocomplete ...

Im experiencing the same issue on Ubuntu 20.04 + OpenJDK-11
Here: 
No Javadoc on typing:
![2021-01-18_12-43](https://user-images.githubusercontent.com/24926168/104948185-e65ac780-598a-11eb-98b1-0f4600400529.png)


But it actualy shows on hover:
![image](https://user-images.githubusercontent.com/24926168/104948261-05f1f000-598b-11eb-9ed7-4f0ea8113869.png)

This is really annoying since you have to type the different options for a method for you to see the documentation.

**Comment by Ortega-Dan** [comment]
I confirm here that in my case it just got fixed by disabling intellicode's extension ...
![image](https://user-images.githubusercontent.com/24926168/104948919-122a7d00-598c-11eb-9995-15671aea24d4.png)

it is working now ..
![image](https://user-images.githubusercontent.com/24926168/104949005-40a85800-598c-11eb-9336-e31c2d6608bc.png)

**Comment by ahmedhamdy90** [comment]
@Ortega-Dan Yes, it should be fixed in the next release of the IntelliCode. MicrosoftDocs/intellicode#263

**Comment by fbricon** [comment]
Closing as caused by IntelliCode.
